components/member-profile/contribution/index.js

```diff
@@ -10,11 +11,70 @@ const renderPRLinks = (prList) =>
 
 const Contribution = ({ contribution, fullName, imageLink, devUser }) => {
   const {
-    task: { title, startedOn, endsOn, status, purpose, featureUrl },
+    task: { featureUrl },
+    prList
+  } = contribution;
+  const url = featureUrl || prList[0]['url'];
+  const gotoUrl = () => url && window.open(url, '_blank');
+  const urlObj = url && new URL(url);
+  const card = () => (
+    <ContributionCard
+      contribution={contribution}
+      fullName={fullName}
+      imageLink={imageLink}
+      devUser={devUser}
+      url={url}
+      urlObj={urlObj}
+    />
+  );
+  const renderFeatureCard = () => {
+    if (urlObj?.host === 'members.realdevsquad.com') {
+      return (
+        <Link href={urlObj.pathname}>
+          <div className={url && classNames.contributionCard}>{card()}</div>
+        </Link>
+      );
+    } else {
+      return (
+        <div
+          className={url && classNames.contributionCard}
+          onClick={gotoUrl}
+          onKeyDown={gotoUrl}
+          aria-hidden="true">
+          {card()}
+        </div>
+      );
+    }
+  };
+
+  return renderFeatureCard();
+};
+
+const ContributionCard = ({ contribution, fullName, imageLink, devUser, url, urlObj }) => {
+  const {
+    task: { title, startedOn, endsOn, status, purpose },
     prList
   } = contribution;
   const isTitleAvailable = title ? true : false;
   const featureTitle = isTitleAvailable ? title : prList[0].title;
+  const renderFeatureUrl = (url, urlObj) => {
+    if (url) {
+      if (urlObj.host === 'members.realdevsquad.com') {
+        return (
+          <Link href={urlObj.pathname} onClick={(e) => e.stopPropagation()}>
+            Check out this feature in action
+          </Link>
+        );
+      } else {
+        return (
+          <a href={url} onClick={(e) => e.stopPropagation()} target="_blank" rel="noreferrer">
+            Check out this feature in action
+          </a>
+        );
+      }
```

Comment:
Few pointers here on the `if-else`

1. Move `members.realdevsquad.com` to a constat file and then use it here
2. Check [this](https://github.com/Real-Dev-Squad/website-members/pull/226/files#r695047922) comment